src/parse/rules/sh_rules.build_defs

```diff
@@ -19,29 +19,32 @@ exit 0\n\
 %(tarball_boundary)s"""
 
 
-def sh_library(name, src, deps=None, visibility=None, link=True):
+def sh_library(name, src, deps=None, visibility=None, link=True, labels=None):
     """Generates a shell script binary, essentially just the given source.
 
     Note that these are individually executable so can only have one source file each.
     This is a bit tedious and would be nice to improve sometime.
 
     Args:
-      name (str): Name of the rule
-      src (str): Source file for the rule
-      deps (list): Dependencies of this rule
+      name (str): Name of the rule.
+      src (str): Source file for the rule.
+      deps (list): Dependencies of this rule.
       visibility (list): Visibility declaration of the rule.
       link (bool): Deprecated, has no effect.
+      labels (list): List of labels.
     """
+    labels = labels or []
```

Comment:
nit: this line is unnecessary (passing labels=None should be perfectly acceptable)